src/NzbDrone.Core/Applications/Lidarr/LidarrV1Proxy.cs

```diff
@@ -102,7 +102,17 @@ public ValidationFailure TestConnection(LidarrIndexer indexer, LidarrSettings se
 
             try
             {
-                Execute<LidarrIndexer>(request);
+                var version = _httpClient.Post<LidarrIndexer>(request).Headers.GetSingleValue("X-Application-Version");
+
+                if (version == null)
+                {
+                    return new ValidationFailure(string.Empty, "Failed to fetch Lidarr version");
+                }
+
+                if (new Version(version) < new Version("1.0.0.0"))
```

Comment:
first lidarr v1 is [1.0.2.2592](https://github.com/Lidarr/Lidarr/releases/tag/v1.0.2.2592)

else current is [1.1.4.3027](https://github.com/Lidarr/Lidarr/releases/tag/v1.1.4.3027)